### ISSUE TITLE
Major Error - Dupes being detected as 100%, when the file name and file length are different!

### ISSUE THREAD

**Issue by PoeticKinetics** [hsoft/dupeguru#434]
Hi. as per the subject line, there appears to be a big issue with the analysis algorithm. Lots of files are being incorrectly listed as a dupe, when many fields don't actually match each other. I didn't check every file for my first few scans, I just went to show Dupes only, and then deleted them. I now feel that I have most likely deleted some files incorrectly due to this error :(

I've attached a screen shot of just one of the examples. Please check the dupe's listed. If you could look into this error and see if it can be rectified, then I'm happy to keep using the program. It's a great concept, but unfortunately with the current reliability issue, it's a bit too hands on to double check each file manually.   Cheers.

![screen shot 2017-08-08 at 9 29 34 pm](https://user-images.githubusercontent.com/30834922/29074612-911d8b88-7c82-11e7-96e7-2496a21ffeb9.png)

**Comment by hsoft** [comment]
What version are you using? What scanning preferences?

**Comment by PoeticKinetics** [comment]
Hi, cheers for your reply. 

I'm using DupeGuru Music Edition version 6.7.0, which I only just downloaded recently. Have I downloaded an old version? _This indicates copyright of the program as 2013._

I've checked the preferences and the scan mode is set to Tags, should I have this set to Filename instead all of the time?  

It seems odd that even if it was scanning the tags, that it would show a 100% match? - I assume that this represents a full text match across all checked scan fields, rather just one of the fields such as comments? - Or does this then come down to the ' filter hardness' setting, which the default is 80, should this be 100?

What hardness setting should I use for Filenames scanning? I would want to detect, files name differences such as 'copy' and '- 1', and maybe the same file names, but one has a website appended to the end, however if two songs have the same artist - title, however one of them is a remix. I wouldn't want this detected as a dupe, even though based on the string, it could be similar to a website that is appended to the end.

~ Cheers
